_posts/2019-02-06-OAS-3.md

```diff
@@ -87,7 +87,7 @@ or providing valuable feedback in our [issue
 tracker](https://github.com/apiaryio/api-elements.js/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aopenapi3).
 
 You can track the [current status of supported OAS 3
-functionality](https://github.com/apiaryio/api-elements.js/blob/master/packages/fury-adapter-oas3-parser/STATUS.md),
+functionality](https://github.com/apiaryio/api-elements.js/blob/master/packages/openapi3-parser/STATUS.md),
 we are also publishing a [list of the changes we make to our
 parser](https://github.com/apiaryio/api-elements.js/blob/master/packages/fury-adapter-oas3-parser/CHANGELOG.md)
```

Comment:
```suggestion
parser](https://github.com/apiaryio/api-elements.js/blob/master/packages/openapi3-parser/CHANGELOG.md)
```

Reply:
There's also `fury-adapter-oas3-parser` on line 85, but GitHub won't let me suggest there as its too far out of the diff view.